### PR TITLE
Bug 1857809: Resource Quota title is duplicated

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @rhamilto

### CODE REVIEW

**Comment by rhamilto** [comment]
This change results in the standalone Resource Quotas page losing its title.  
Screen Shot 2020-07-17 at 7 23 44 AM

File: frontend/public/components/resource-quota.jsx [hunk 1]
```diff
@@ -428,6 +428,7 @@ export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(({ namespace,
       title="Resource Quotas"
       rowFilters={rowFilters}
       mock={mock}
+      showTitle={false}
```
Comment (rhamilto) [inline frontend/public/components/resource-quota.jsx:431]:
```suggestion
      showTitle={showTitle}
```

Then change 387 as follows:
```
export const ResourceQuotasPage = connectToFlags(FLAGS.OPENSHIFT)(
  ({ namespace, flags, mock, showTitle = true }) => {
```

**Comment by jhadvig** [comment]
@rhamilto comment addressed. Thanks!

**Pull request merged**